StreamChatClient_v3/ChatClient.swift

```diff
@@ -233,6 +179,54 @@ extension Client {
         
         return headers
     }
+    
+    /// Creates a new instance of Stream Chat `Client`.
+    ///
+    /// - Parameters:
+    ///   - currentUser: The user instance representing the current user of the chat.
+    ///   - config: The config object for the `Client`. See `ChatClientConfig` for all configuration options.
+    public convenience init(currentUser: UserModel<ExtraData.User>, config: ChatClientConfig) {
+        // All production workers
+        let workerBuilders: [WorkerBuilder] = [
+            MessageSender.init,
+            ChannelEventsHandler<ExtraData>.init
+        ]
+        
+        self.init(currentUser: currentUser,
+                  config: config,
+                  workerBuilders: workerBuilders,
+                  environment: .init())
+    }
+    
+    /// Creates a new instance of Stream Chat `Client`.
+    ///
+    /// - Parameters:
+    ///   - currentUser: The user instance representing the current user of the chat.
+    ///   - config: The config object for the `Client`.
+    ///   - workerBuilders: An array of worker builders the `Client` instance will instantiate and run in the background
+    ///   for the whole duration of its lifetime.
+    ///   - environment: An object with all external dependencies the new `Client` instance should use.
+    init(currentUser: UserModel<ExtraData.User>,
+         config: ChatClientConfig,
+         workerBuilders: [WorkerBuilder],
+         environment: Environment) {
+        
+        self.config = config
+        self.currentUser = currentUser
+        self.environment = environment
+        
+        backgroundWorkers = workerBuilders.map { builder in
+            builder(self.persistentContainer, self.webSocketClient, self.apiClient)
+        }
+    }
+}
+
+// MARK: ========= TEMPORARY!
+
+extension Client {
+    var token: String {
+        "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoiYnJva2VuLXdhdGVyZmFsbC01In0.d1xKTlD_D0G-VsBoDBNbaLjO-2XWNA8rlTm4ru4sMHg"
+    }
```

Comment:
It's better to mark this temporary stuff as `FIXME:` instead of `MARK: ==== ...`

Reply:
This has been there since the first v3 commit. I'm removing this part in CIS-164 anyway, so I'll keep it.